packages/editor/src/client/routes/pageEditor.tsx

```diff
@@ -228,7 +228,9 @@ export function PageEditor({id}: PageEditorProps) {
     }
 
     setChanged(false)
-    Alert.success(t('pageEditor.overview.pagePublished'), 2000)
+    publishData === new Date() || publishDate < new Date()
```

Comment:
I think you can use smaller or equal for the comparison instead of comparing or smaller. `publishDate <= new Date()`.

Also i think you have a typo. The first argument is publishDat**a** instead of publishDat**e**.